src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -567,6 +567,39 @@ def list_unix_file_systems(self, file_path_name=None, file_system_name=None):
         response_json = self.request_handler.perform_request("GET", custom_args, expected_code=[200])
         return response_json
 
+    def delete_migrated_data_set(self, dataset_name: str, wait=False, purge=False):
```

Comment:
I think the order of those arguments should be swapped, first `purge` and then `wait` as last. If you do this change, review your parametrized test that it is still structured correctly.

Reply:
Ok, will swap them and refactor the tests.